collector/src/main/java/com/bitmovin/analytics/BitmovinAdAnalytics.kt

```diff
@@ -4,32 +4,39 @@ import com.bitmovin.analytics.ads.*
 import com.bitmovin.analytics.data.AdSample
 import com.bitmovin.analytics.data.AdEventData
 import com.bitmovin.analytics.utils.Util
-import java.util.Collections.min
 
 class BitmovinAdAnalytics(var analytics: BitmovinAnalytics) {
     private var activeAdBreak: AdBreak? = null
     private var activeAdSample: AdSample? = null
     private var adPodPosition: Int = 0
     private var adStartupTimestamp: Long? = null
-    private var currentTime: Long? = null
+    private var _currentTime: Long? = null
```

Comment:
What you are doing here can be achieved in pure kotlin with a backing `field`:
https://kotlinlang.org/docs/reference/properties.html#backing-fields

The `_currentTime` looks really weird